schema.sql

```diff
@@ -83,4 +83,13 @@ CREATE TABLE visits (
     FOREIGN KEY (animal_id) REFERENCES animals (id) ON DELETE CASCADE,
```

Comment:
* `SELECT COUNT(*) FROM visits where animal_id = 4;`
* `SELECT * FROM visits where vet_id = 2;`
* `SELECT * FROM owners where email = 'owner_18327@mail.com';`
Please add a pair of screenshots in the PR description with the results of the `EXPLAIN ANALYZE` command for each of the above queries (in total it should be 6 screenshots).